process_report/processors/prepayment_processor.py

```diff
@@ -22,11 +22,24 @@ class PrepaymentProcessor(discount_processor.DiscountProcessor):
     def PREPAY_DEBITS_S3_BACKUP_FILEPATH(self):
         return f"Prepay/Archive/prepay_debits {util.get_iso8601_time()}.csv"
 
+    @property
```

Comment:
I usually try to have the following order for things defined in a class.

1. attributes
2. properties
3. methods

This makes it easier to find out where things are defined, so please move the properties after the attributes.